library/ceph_volume.py

```diff
@@ -153,16 +177,12 @@ def create_osd(module):
     wal_vg = module.params.get('wal_vg', None)
     crush_device_class = module.params.get('crush_device_class', None)
     dmcrypt = module.params['dmcrypt']
+    containerized = module.params.get('containerized')
+    subcommand = "create"
```

Comment:
Shouldn't this be ``subcommand = "prepare"``?

Reply:
@andrewschoen thanks for the review even though this wasn't ready for review :), will address your comments soon.